server/routes/index.ts

```diff
@@ -51,10 +53,38 @@ export default function routes(services: Services): Router {
   use('/add-a-bank-account', addBankAccountRouter)
   use('/health-status', healthRouter)
   use('/licence-image', licenceImageRouter)
-  use('/status-update/:selectedPathway', (req: Request, res: Response, next) => {
+  use('/status-update/:selectedPathway', async (req: Request, res: Response, next) => {
     const { prisonerData } = req
     const { selectedPathway } = req.params
-    res.render('pages/status-update', { prisonerData, selectedPathway })
+    const { state } = req.query
+    const token = res.locals?.user?.token
+    let updateSuccessful = false
+    if (state) {
+      try {
+        const pathRequest = await fetch(
+          `https://resettlement-passport-api-dev.hmpps.service.justice.gov.uk/resettlement-passport/prisoner/${prisonerData.personalDetails.prisonerNumber}/pathway`,
```

Comment:
Use RPClient instead of hardcoding URL

Reply:
Currently the ```get``` method in the rpClient doesn't support a 3rd argument for the ```options``` object in the fetch API.

Reply:
Can you add that in or create a task to do that? We can't go to PreProd until we can dynamically pick up which environment (URL) to use for the API